testgrid/config/config.yaml

```diff
@@ -2674,78 +2581,35 @@ dashboards:
   - name: periodic-bazel-test-1.7
     test_group_name: periodic-kubernetes-bazel-test-1-7
 
+# These are the release *blocking* tests.  These provide a valid binary signal
+# to gate releases (alpha, beta, official).
+# This list is used by kubernetes/release/lib/releaselib.sh to provide no/nogo
+# signal for releases.
+# The first test/suite in the list is the primary suite for release purposes
 - name: release-1.7-blocking
   dashboard_tab:
-  - name: build-1.7
```

Comment:
Thanks for the replies, @caesarxuchao!  That does seem a reasonable primary test suite.  The current list of suites hasn't been updated in some months and many new suites have been created, so a full audit is due.  If noone objects, we'll make the build job primary.